market/demos/excel-connector/README.md

```diff
@@ -0,0 +1,12 @@
+# Excel Connector Demo
+Axon Ivy's Excel Connector Demo nicely showcases how easily you can import and export data records. The solution illustrates how:
```

Comment:
also here: not just a demo: This connector actually imports and exports Excel datasets into your process datas.